Week1/homework/js-exercises/8th exercise/type.js

```diff
@@ -0,0 +1,41 @@
+//Write a program that checks the data types of two variables and logs to the console SAME TYPE if they are the same type. If they are different types log Not the same....
+
+//Declare 4 variables: 2 must be strings and 2 must be objects
+var a = "apple";
+var b = "orange";
+var c = {
+  firstName: "Mario",
+  lastName: "Ramirez"
+};
+
+var d = {
+  city: "Amsterdam",
+  country: "The Netherlands"
+};
+//Create 8 conditional statements, where for each you check if the data type of one variable is the same as the other
+
+//Find out how to check the type of a variable
+console.log(typeof a);
+console.log(typeof b);
+console.log(typeof c);
+console.log(typeof d);
+
+//Write 2 console.log statements to log the type of 2 variables, each with a different data type
+console.log(a);
+console.log(c);
+//Now compare the types of your different variables with one another
+
+//Log Not the same... when the types are different
+/*Here's an incomplete example of how it could look:
+
+// Declare all variables
+let x = 9;
+let y = 67;
+
+// Check data type
+console.log(...);
+
+// Check if data type is the same
+if (...) {
+    console.log('SAME TYPE');
+}*/
```

Comment:
I think you missed the last part of comparing the data types of variables you created using the `if-else` conditions. Also, please use `const` instead of `var` here. 🙂